src/reference/config/bindings/proxy/.partials/routes.md

```diff
@@ -0,0 +1,140 @@
+### routes
+
+> `array` of `object`
+
+Conditional `proxy`-specific routes.
+
+```yaml
+routes:
+  - when:
+      - transport: stream
+        family: inet4
+        destination:
+          port: 443
+    exit: tls_server
+```
+
+#### routes[].guarded
+
+> `object` as named map of `string:string` `array`
+
+List of roles required by each named guard to authorize this route.
+
+```yaml
+routes:
+  - guarded:
+      my_guard:
+        - read:items
+```
```

Comment:
Even though this is officially supported by the syntax, there is typically no useful guard check to make as the proxy protocol tends to kick in so early in the pipeline that there isn't much context established yet to apply a guard.

Reply:
If no guards work on the proxy, can we add the `guarded: false` to the schema? The reference section should be an accurate representation of the Schema Interface so it would feel weird to technically allow something in case we want to do it but then have to reference to it in the docs. 

IMO, if something in the reference section has no practical examples anywhere else in the docs or other repos, then it is the same thing as saying, "You probably don't need to worry about this setting even though it exists." This is also why it is important to have practical examples for almost everything 😆

Reply:
Actually, I was wrong, this is supported generically on routes, so we are all good. 👍